package.json

```
@@ -24,5 +24,11 @@
   "dependencies": {
     "@stencil/core": "^2.0.1"
   },
+  "devDependencies": {
+    "@types/jest": "^26.0.24",
+    "jest": "^26.6.3",
```

Ah, it looks like we need to add an explicit dependency on `jest-cli` too - I thought that wouldn't be needed as it's a subpackage of `jest`, my mistake! Would you mind adding one more dependency?
```suggestion
    "jest": "^26.6.3",
    "jest-cli": "^26.6.3",
```